language/move-lang/src/expansion/translate.rs

```diff
@@ -309,8 +309,12 @@ fn script_(context: &mut Context, pscript: P::Script) -> E::Script {
     }
 
     let (function_name, function) = function_(context, pfunction);
-    if let FunctionVisibility::Public(loc) = &function.visibility {
-        let msg = "Extraneous 'public' modifier. Script functions are always public";
+    if let FunctionVisibility::Public(loc) | FunctionVisibility::Script(loc) = &function.visibility
```

Comment:
Let's just switch this to an exhaustive match, now that we are expecting more visibilities.

When I wrote this originally, I was so confident we would just have public+internal :P